rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :customer
```

Comment:
- [ ] Getters and Setters for @book and @person instance variables should be present, kindly change `customer` to `person`